imgui-gfx-renderer/src/lib.rs

```diff
@@ -214,10 +254,10 @@ impl<R: Resources> Renderer<R> {
 
         self.bundle.slice.start = 0;
         for cmd in draw_list.cmd_buffer {
-            // TODO: check cmd.texture_id
+            let tex = self.textures.get(cmd.texture_id.into()).unwrap_or(&self.backup_texture);
```

Comment:
I think it would be better to return an error if an invalid id is seen. See my other comment in the glium renderer